lib/sw_indexer_engine.rb

```diff
@@ -14,7 +14,7 @@ def index(druid, targets = nil)
     # If a catkey exists in the purl_model, stop processing the druid and leave
     # the method because access to the digital object will be provided by an 856
     # in the corresponding MARC record
-    return if purl_model.catkey
+    return unless purl_model.catkey
```

Comment:
Since this was confusing enough before, maybe we should rewrite it to be clearer?